commander/src/bootstrapping/commands/transaction/create.ts

```diff
@@ -300,18 +291,23 @@ export abstract class CreateCommand extends Command {
 		this._dataPath = flags['data-path'] ?? getDefaultPath(this.config.pjson.name);
 
 		if (flags.offline) {
-			const { genesisBlock, config } = await getGenesisBlockAndConfig(flags.network);
-			const app = this.getApplication(genesisBlock, config);
+			const app = this.getApplication({}, {});
 			this._schema = app.getSchema();
+
 			transactionObject = await createTransactionOffline(
 				args as Args,
 				flags,
 				this._schema,
 				incompleteTransaction,
 			);
 		} else {
-			this._client = await getApiClient(flags['data-path'] as string, this.config.pjson.name);
+			if (!isApplicationRunning(this._dataPath)) {
+				throw new Error(`Application at data path ${this._dataPath} is not running.`);
+			}
```

Comment:
`await getApiClient` already checks for app running, why do we need this extra check?